README.md

```diff
@@ -46,14 +38,11 @@ which the qubits have different values.
 In the results, you should notice that the solutions (0 1) and (1 0) have 
 lower energy than the solutions (0 0) and (1 1).
 
-Exercise 3 
-----------
-
-Run the command
+## Exercise 3 
 
-.. code-block:: bash
+To run the exercise 3 demo from the command-line, type the command:
 
-  python two_implies.py
+```python two_implies.py```
 
 Here is a definition of the implies function, from Wikipedia:
```

Comment:
Can you add a link here? We can make the word "Wikipedia" clickable:

`[Wikipedia](www.linktoarticlehere.com)`